tgui/packages/tgui/interfaces/BookBinder.js

```diff
@@ -0,0 +1,109 @@
+import { useBackend, useLocalState } from '../backend';
+import {
+  Box,
+  Button,
+  Dropdown,
+  Flex,
+  Icon,
+  LabeledList,
+  Section,
+} from '../components';
+import { Window } from '../layouts';
+import { ComplexModal, modalOpen} from './common/ComplexModal';
+import { FlexItem } from '../components/Flex';
+
+export const BookBinder = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    selectedbook,
+    book_categories,
+  } = data;
+
+  let categoryMap = []
+  {book_categories.map(category => (
+    categoryMap[category.description] = category.category_id
+  ))}
+
+  return (
+    <Window resizable>
+      <ComplexModal/>
+      <Window.Content scrollable className="Layout__content--flexColumn">
+      <Section title="Book Binder">
+      <Box fontSize="1.2rem" bold>
```

Comment:
TGUI lint probably wants these spaced properly

Reply:
I'll get this cleaned up, I forgot to go through and fix indentation on all my js files

Reply:
fixed